fud/icarus/icarus.py

```diff
@@ -113,6 +115,7 @@ def compile_with_iverilog(
                 cmd.format(
                     input_path=str(input_path),
                     exec_path=f"{tmpdir.name}/{self.object_name}",
+                    include_path="-I " + config["include"],
```

Comment:
Seems like this need to reduce to nothing if `config["include"]` is unset. As in, adding ` if config["include"] else ""`, so that the command-line doesn't have a plain `-I` on it if there is no include path specified.